### DISCUSSION TITLE
Terms without definitions

### DISCUSSION THREAD

**Discussion by ValWood** [monarch-initiative/mondo#3874] [Bigger project]
We now have a lot of terms without definitions which had definitions previously from DO
see 

[mondo_terms_no_definition.tsv (1).txt](https://github.com/monarch-initiative/mondo/files/4592722/mondo_terms_no_definition.tsv.1.txt)

**Comment by nicolevasilevsky** [comment]
@cmungall do we want to add the definitions from DO?

**Comment by ValWood** [comment]
These defs were really useful.
I also liked the fact that most mentioned the causative gene,  which made it clear that we had the fission yeast assignment  correct.

**Comment by nicolevasilevsky** [comment]
@ValWood were all the terms in the list above in DO (and defined in DO)? I did some spot checking of some terms and I didn't see all of the terms in DO. Thanks!

**Comment by ValWood** [comment]
Oh sorry no. The ones I mentioned originally (I can't remember which ticket)  had defs in DO when I looked. This is just a list of everything without defs. I don't think we can easily now get just the list without MONDO defs but with DO defs, but I can ask @kimrutherford if it would be really useful

**Comment by nicolevasilevsky** [comment]
Hi Val - no need to get the list without MONDO defs but with DO defs. I talked to Chris today and we have a strategy.

We'll try to pull in any defs from OMIM where we can, and I'll review the list to determine other sources of defs. This will take me some time though, I hope that's okay.

I added this list to a spreadsheet [here](https://docs.google.com/spreadsheets/d/1a5thoV8kQ3tShdmiRaWB1ODEfxtKWSh8f3yAPltKHrI/edit#gid=416513835).

**Comment by ValWood** [comment]
No hurry for me. This sounds like a long term task!

val

**Comment by nicolevasilevsky** [comment]
@ShahimEssaid your action item is:
- [ ] transfer text defs from OMIM into Mondo w/ provenance 
Chris described this workflow in the MOSS-Cide mtg
